ihor.bohdan/second/task 1/tamagochi.rb

```diff
@@ -0,0 +1,88 @@
+class Tamagochi
+  attr_accessor :health
```

Comment:
As you decided to declare attr_accessor you need to actually use it. For example line 8 should be `self.health = health`

Reply:
done